engine/src/test/java/io/camunda/zeebe/engine/state/BanInstanceTest.java

```diff
@@ -235,6 +235,10 @@ public void shouldBanInstance() {
 
   private final class Value extends UnifiedRecordValue implements ProcessInstanceRelated {
 
+    public Value() {
+      super(10);
```

Comment:
:question: @oleschoenburg it's not very clear to me why we set the number of declared properties to 10 here. Can you clarify?